src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -215,12 +228,18 @@ def register_extra_cli_argument(command, dest, **kwargs):
     '''
     _cli_extra_argument_registry[command][dest] = CliCommandArgument(dest, **kwargs)
 
+def cli_command_with_handler(module_name, name, operation, client_factory=None, transform=None, table_transformer=None):
```

Comment:
That was temporary so the 'old way' will still work as I haven't changed everything over.
Once done, we'll just have cli_command() that behaves in the new way.